main.tf

```diff
@@ -0,0 +1,141 @@
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-data"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "4d57f83ca4d3a78a44fb36d1dcf0d23983fa44f5"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2022-05-18 07:08:06"
+    git_last_modified_by = "nimrod@bridgecrew.io"
+    git_modifiers        = "34870196+LironElbaz/nimrod/nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "0874007d-903a-4b4c-945f-c9c233e13243"
+  })
+}
+
+resource "aws_s3_bucket_object" "data_object" {
+  bucket = aws_s3_bucket.data.id
+  key    = "customer-master.xlsx"
+  source = "resources/customer-master.xlsx"
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-customer-master"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "a7f01cc7-63c2-41a8-8555-6665e5e39a64"
+  })
+}
+
+resource "aws_s3_bucket" "financials" {
+  # bucket is not encrypted
+  # bucket does not have access logs
+  # bucket does not have versioning
+  bucket        = "${local.resource_prefix.value}-financials"
+  acl           = "private"
+  force_destroy = true
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-financials"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "0e012640-b597-4e5d-9378-d4b584aea913"
+  })
+
+}
+
+resource "aws_s3_bucket" "operations" {
+  # bucket is not encrypted
+  # bucket does not have access logs
+  bucket = "${local.resource_prefix.value}-operations"
+  acl    = "private"
+  versioning {
+    enabled = true
+  }
+  force_destroy = true
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-operations"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "29efcf7b-22a8-4bd6-8e14-1f55b3a2d743"
+  })
+}
+
+resource "aws_s3_bucket" "data_science" {
```

Comment:
<details><summary>
<img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsSeverityIcons/low_new.svg" alt="LOW" width="24px" align="center">&nbsp;&nbsp;<b style="color:black" >Ensure S3 Bucket has public access blocks</b>
            <br>
            &nbsp;&nbsp;&nbsp;&nbsp;Resource: <a href="https://github.com/marynaKK/terragoat/blob/marynaKK-patch-1/main.tf#L89-L111">aws_s3_bucket.data_science</a> | Bridgecrew ID: <code>BC_AWS_NETWORKING_52</code> | Checkov ID: <a href="https://docs.prismacloud.io/en/enterprise-edition/policy-reference/aws-policies/aws-networking-policies/s3-bucket-should-have-public-access-blocks-defaults-to-false-if-the-public-access-block-is-not-attached">CKV2_AWS_6</a>
            <br>
            
</summary>
<h4>How to Fix</h4>

```tf
resource "aws_s3_bucket" "bucket_good_1" {
  bucket = "bucket_good"
}


+ resource "aws_s3_bucket_public_access_block" "access_good_1" {
+   bucket = aws_s3_bucket.bucket_good_1.id
+ 
+   block_public_acls   = true
+   block_public_policy = true
+ }
```

<h4>Description</h4>


When you create an S3 bucket, it is good practice to set the additional resource  *aws_s3_bucket_public_access_block* to ensure the bucket is never accidentally public.
We recommend you ensure S3 bucket has public access blocks.
If the public access block is not attached it defaults to False.

</details>
<p><img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsBotLogos/bc.svg" alt="Bridgecrew" width="18px">&nbsp;<small>Comment by Bridgecrew Bot</small></p>